shuup/front/apps/product_filter/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
This file is not needed, but for sure tests are welcome under `shuup_tests` or  `shuup_tests.front`.